### PR TITLE
allow append datapoint to a dataset

### PR DESCRIPTION
**Pull request by grant-Kolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: kolena/dataset/dataset.py [hunk 1]
```diff
@@ -325,6 +326,9 @@ def upload_dataset(
         within a dataset. When unspecified, a suitable value is inferred from the columns of the provided `df`. Note
         that `id_fields` must be hashable.
     :param commit_tags: Optionally specify a list of tags to associate with the dataset commit.
+    :param append_only: If True, the existing datapoint in the dataset will not be modified,
```
Comment (nankolena) [inline kolena/dataset/dataset.py:329]: nit - `existing datapoints` plural

File: kolena/dataset/dataset.py [hunk 2]
```diff
@@ -325,6 +326,9 @@ def upload_dataset(
...
+    and only new datapoints from the input dataframe will be added. If false, all datapoint in the dataset will
```
Comment (nankolena) [inline kolena/dataset/dataset.py]: nit - `If False` with capital F, and `all datapoints` plural

**Review by nankolena — COMMENTED** [review]
Can we add a quick integ test on this?

**Comment by grant-Kolena** [comment]
> Can we add a quick integ test on this?

done

**Review by nankolena — APPROVED** [review]

**Review by yifanwu-kolena — APPROVED** [review]

**Pull request merged**